include/tbl/oc_flags.h

```diff
@@ -30,7 +30,8 @@
 
 /*lint -save -e525 -e539 */
 
-OC_FLAG(BUSY,		busy,		(1<<1))		//lint !e835
+OC_FLAG(WITHDRAW,	withdraw,	(1<<0))		//lint !e835
```

Comment:
Should this be named `WITHDRAWN` instead? When set I believe the past tense is the right one, as in it has already been withdrawn.

Reply:
Good point.